blueocean-dashboard/src/main/less/connection-lost.less

```diff
@@ -0,0 +1,30 @@
+.errorscreen {
+    &.connection-lost {
+        background-color: @errorscreen-bg-color;
+    }
+    &.connection-ok {
+        background-color: @blueocean-blue;
+    }
+
+    &.connection-lost, &.connection-ok {
+        /*
+         * Lets ensure that the lost and ok messages are the same
+         * width Vs jumpy message box resizes.
+         */
+        .message-box {
+            min-width: 30em;
+            max-width: 30em;
+        }
+
+        /*
+         * For some reason, we needed to set the font-weight on the
+         * h3 used in the errorscreen when disconnected. If not, it
+         * appears more bold than it does if the same screen is
+         * shown when connected, plus when compared to the 404 screen.
+         * Weird :(
+         */
+        h3 {
+            font-weight: 400;
+        }
```

Comment:
So if I remove this `h3` rule and unconditionally show the error connection screen (i.e. when the connection is actually fine) I see the font-weight as expected (i.e. the same as for the 404 screen) ...

![screenshot 2017-01-23 12 14 18](https://cloud.githubusercontent.com/assets/429311/22204314/3d1ba7c2-e169-11e6-81ee-e3bc8d18d072.png)

If I remove the unconditional behaviour and only show that same screen when there is no connection, notice how the font-weight is heavier ....

![screenshot 2017-01-23 12 30 43](https://cloud.githubusercontent.com/assets/429311/22204346/6a736980-e169-11e6-8c48-b3a8555f3cca.png)

Only difference is that there's a connection for the first, while there's no connection for the second. Wtf ??